pkg/auth/handler/webapp/authflowv2/login.go

```diff
@@ -89,6 +100,20 @@ func (h *AuthflowV2LoginHandler) ServeHTTP(w http.ResponseWriter, r *http.Reques
 		return
 	}
 
+	if r.URL.Query().Get(webapp.PreviewQueryKey) == webapp.PreviewModeInline {
```

Comment:
Abstract this expression `r.URL.Query().Get(webapp.PreviewQueryKey) == webapp.PreviewModeInline` to a function

```go
func IsPreViewModeInline(r) bool
```